changelogs/changelog.yaml

```diff
@@ -108,3 +108,31 @@ releases:
         name: eos
         namespace: null
     release_date: '2020-06-23'
+  1.0.2:
+    changes:
+      bugfixes:
+      - Fixes mismatch in documentation and code for using eos_lag_interfaces where
+        the code required 'Port-Channel\d.*:' but the docs did not document this.
+        The module now supports both 'Port-Channel\d.*:' and '\d.*:'.
+      - Make `src`, `backup` and `backup_options` in eos_config work when module alias
+        is used (https://github.com/ansible-collections/arista.eos/pull/85).
+      minor_changes:
+      - Add changelogs for 1.0.0 release (https://github.com/ansible-collections/arista.eos/pull/62)
+      - Add round trip testcases to the 2.9 resource modules.
+      - Add unit testcases to the eos_l3_interfaces resource modules.
+      - Add unit testcases to the eos_lag_interfaces resource modules.
+      - Sorted the list of params of ip address before forming the tuple.
+      - Updated docs.
+      release_summary:
```

Comment:
I believe the all the changelogs would go under 1.0.1 and the 1.0.2 would just have a release_summary entry that mentions that it's a re-release of 1.0.1 with updated changelog.